src/components/pages/donation_form/DonorSummary.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+	<div class="donor-summary" v-if="hasData">
```

Comment:
Similar to the last comment you can format this better for screen readers like this:

```html
<h3>{{ $t('donation_summary_contact_data_header') }}</h3>
<p class="donor-summary" v-if="hasData">
	<template v-if="name">{{ name }}<br/></template>
	<template v-if="address.street">{{ cleanedStreetAddress }}<br/></template>
	<template v-if="address.postcode || address.city">
		{{ address.postcode }} {{ address.city }}<br/>
	</template>
	<template v-if="country">{{ country }}<br/></template>
	<template v-if="email">{{ email }}<br/></template>
</p>
```

The address then becomes a single block for screen readers to read rather than the person having to skip through div by div

Reply:
@Abban I need to put `<h3>{{ $t('donation_summary_contact_data_header') }}</h3>` inside a parent block (maybe <div>) which has the condition `v-if="hasData"`.

If I do it as you suggested then the headline will always be shown even if the user has not entered any contact data.

Could I use <div> as the parent of <h3>? Because having <p> as parent of <h3> is not allowed (IDE is complaining)

Reply:
Yep you shouldn't wrap it all in a p tag. You can do 3 things:

- Add a v-if to the h3
- Wrap the whole block in a div
- Wrap the block in a template (It might be an issue if a template tag is at the top level or with nested template tags, but I think this might work)